constance/templates/admin/constance/includes/results_list.html

```diff
@@ -20,7 +20,10 @@
             {{ item.form_field.errors }}
             {{ item.form_field }}
             <br>
-            <a href="#" onClick="document.getElementById('{{ item.form_field.auto_id }}').value = '{{ item.default|escapejs }}'; return false;">Reset to default</a>
+            <a href="#" onClick="
+              document.getElementById('{{ item.form_field.auto_id }}').{% if item.form_field_name == 'CheckboxInput' %}checked =
+                  {% if item.default == 'True' %} true {% else %} false {% endif %}
```

Comment:
@rvernica Ah, I missed that, good point. Then let's follow @therefromhere's idea and pass the raw default value as well to make the check. I can't come up with a better name, `raw_default` sounds good though.